src/super_context.ml

```diff
@@ -848,6 +854,10 @@ module PP = struct
     | [key] ->
       let ppx_dir = Path.relative sctx.ppx_dir key in
       let exe = Path.relative ppx_dir "ppx.exe" in
+      let (key, scope) =
+        match String.rsplit2 key ~on:'@' with
+        | None -> (key, Scope.empty)
```

Comment:
In the end, that's the same as before given that we just do `scope.root` after, right?

Reply:
Yeah, because I'm still inserting the wrong scope under `""`. OK We definitely need the more explicit scope type before we're more confident about it.